pstress/pstress-run-PXC57.conf

```diff
@@ -92,7 +92,7 @@ PXC_MAX_NR_OF_RND_OPTS_TO_ADD=4
 # Save only trials that generate a core file (good for initial few runs where  #
 # there are lot of crashes/asserts)                                            #
 ################################################################################
-SAVE_TRIALS_WITH_CORE_OR_VALGRIND_ONLY=0
+SAVE_TRIALS_WITH_CORE=0
```

Comment:
The variable name should be SAVE_TRIALS_WITH_CORE_ONLY as suggested in my previous comment.
This variable if enabled will save only the trials which has core files and delete others.